src/fbwndgfs.fd/CMakeLists.txt

```
@@ -7,11 +7,7 @@ if(CMAKE_Fortran_COMPILER_ID MATCHES "^(Intel)$")
 elseif(CMAKE_Fortran_COMPILER_ID MATCHES "^(GNU)$")
   set(CMAKE_Fortran_FLAGS "${CMAKE_Fortran_FLAGS} -fdefault-real-8 -fdefault-integer-8 -fconvert=big-endian")
 endif()
-#Parse w3emc version numbers
-string(REPLACE "." ";" W3EMC_VERSIONS ${w3emc_VERSION})
-list(GET W3EMC_VERSIONS 0 W3EMC_MAJOR)
-list(GET W3EMC_VERSIONS 1 W3EMC_MINOR)
-if(W3EMC_MINOR GREATER_EQUAL 10 AND W3EMC_MAJOR GREATER_EQUAL 2)
+if(w3emc_VERSION VERSION GREATER 2.10)
```

```suggestion
if(w3emc_VERSION VERSION_GREATER_EQUAL 2.10)
```
Apologies.  I made a mistake.